java/statestore/src/test/java/sleeper/statestore/dynamodb/DynamoDBFileReferenceStoreIT.java

```diff
@@ -229,6 +229,7 @@ public void shouldSetFileReadyForGC() throws Exception {
             store.addFile(oldFile);
 
             // When
+            store.atomicallyUpdateJobStatusOfFiles("job1", List.of(oldFile));
             store.atomicallyUpdateFilesToReadyForGCAndCreateNewActiveFiles("root", List.of("oldFile"), List.of(newFile));
```

Comment:
There are some differences between the in-memory, DynamoDB and S3 file reference store tests, eg. here the method signature for metadata split is used instead of the one to apply a compaction job. It might be worth copying and pasting from the in-memory one to the others.

Reply:
I've removed the old method signature and the new method signature is being used everywhere now